index.js

```diff
@@ -172,7 +172,7 @@ function linux (args, sudo) {
     }
   })
 
-  var folder = 'profile-' + proc.pid
+  var folder = 'profile-' + proc.pid + '-' + (new Date()).getTime();
```

Comment:
Date.now() would be preferred here 

Let's talk about putting this behind an option